services/agent/requirements/prod.txt

```diff
@@ -11,6 +11,7 @@
 
 # installs this repo's packages
 simcore-models-library @ ../../packages/models-library
+simcore-common-library @ ../../packages/common-library/
```

Comment:
I think you forgot all the ci.txt and devel.txt files.

Reply:
checked all of them, should be OK now